eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,14 @@
+-SourceToolsList @("policheck","credscan")
```

Comment:
> The most useful parts of TSA config are controlling which tools run (default = all tools run)

@GrabYourPitchforks does that mean we're intentionally limiting to these 3 tools - should we be? this copies our other repos.

Reply:
also, are TsaRepositoryName/TsaCodebaseName (and equivalent) more than opaque strings?

specifically, I'm assuming that mapping goes <issue found> --> <find service tree node from repo that was scanned> --> appear in my S360 view does not rely on these strings having particular values. It uses the OSS repo->service tree mapping.

Reply:
> also, are TsaRepositoryName/TsaCodebaseName (and equivalent) more than opaque strings?
> 
> specifically, I'm assuming that mapping goes `<issue found> --> <find service tree node from repo that was scanned> --> appear in my S360 view` does not rely on these strings having particular values. It uses the OSS repo->service tree mapping.

I think they're arbitrary strings. arcade has "arcade" for both.